src/components/Search/Search.js

```diff
@@ -78,7 +78,9 @@ function Search() {
                       {t(section)}
                     </div>
                     <div className="wkp-search-section-count">
-                      {t('overall_result', { count })}
+                      {count > 1
+                        ? t('overall_results', { count })
+                        : t('overall_result', { count })}
```

Comment:
doc first : https://www.i18next.com/translation-function/plurals

Reply:
Use `overall_result` and ` overall_result_plural`

Reply:
I didnt't know this functionality! It is great :)